.appveyor.yml

```diff
@@ -26,22 +26,15 @@ before_build:
   - "SET PATH=C:\\Python35;C:\\Python35\\Scripts;%PATH%"
   - echo.
   - echo Starting build for %APPVEYOR_REPO_NAME%
-  # Install Vulkan-Headers
-  - mkdir %TEMP%\headers
-  - git clone https://github.com/KhronosGroup/Vulkan-Headers.git %TEMP%\headers\repo
-  - ps: pushd $ENV:TEMP\headers\repo
-  - mkdir build
-  - cd build
-  - cmake -DCMAKE_INSTALL_PREFIX=%TEMP%\headers\install ..
-  - cmake --build . --target install
-  - ps: popd
+  # Install dependencies
+  - python scripts/update_deps.py --dir=external
   # Get Google Test
```

Comment:
What about adding google_test to the known_good json file, pulling from HEAD?

Reply:
I'm inclined to leave it as is for the time being for two reasons:
- Googletest is handled differently than Vulkan-Headers, so we might want to rethink the googletest logic if we do that
- What's there now matches Vulkan-ValidationLayers. If we make a change, we probably want to do it in both places.

But this might still be a good future improvement.